exec/cnex/lib/file_win32.c

```diff
@@ -0,0 +1,245 @@
+#include "util.h" // Include util.h so that TRUE and FALSE are declared before including windows.h
```

Comment:
Why is this necessary? Shouldn't the include files be able to be included in any order?